README.md

```diff
@@ -1,3 +1,31 @@
+# dYdX Fork of Tendermint
+
+This is a lightweight fork of Tendermint. Forked code resides on the default branch.
```

Comment:
Can we clarify what the default branch means here? Does it refer to the latest forked tendermint branch that we are using?